pubtools/_quay/tag_docker.py

```diff
@@ -549,6 +553,7 @@ def copy_tag_sign_images(self, push_item: Any, tag: str, executor: Executor) ->
                     continue
                 outdated_manifests.append((mad.digest, tag, repo))
 
+            set_aws_kms_environment_variables(self.target_settings, "cosign_signer")
```

Comment:
Is the key needed for pre-push signing?

Reply:
It's not, but I thought it would be probably better to just keep setup of aws variables before every signing as it devs then doesn't need to remember and determine whether it's needed or not.